Makefile

```diff
@@ -10,7 +10,7 @@ node_dev:
 node_prod:
 	echo 'Starting Node prod server in Docker Container'
 	docker build --build-arg run_mode_arg=start -t node_prod .
-	docker run -p 8080:8080 -e "run_mode_env=start" node_prod
+	docker run -p 8080:8080 -e "run_mode_env=start" node_prod sh
```

Comment:
Did you want the `sh`  here to be committed?

Reply:
No, I should probably get rid of it before I merge. 

This article was useful.  https://www.freecodecamp.org/news/how-to-enable-live-reload-on-docker-based-applications/

But the ignored files still appear when I remote into the container.  But i don't know if they are actually in the container or a result of the mirroring